docs/integrations/sources/salesforce.md

```diff
@@ -33,48 +34,77 @@ If this is Greek to you, then you are likely not using a sandbox account.
+13. Click "save"
+14. Copy the "Username" field and keep it handy -- you will use it when setting up Airbyte later
+15. Login to the email you set in step 12c above and verify your new Salesforce account user. You'll need to set a password as part of this process. Keep this password handy. 
+16. With the username and password, you should be ready to setup the Salesforce connector.  
+    
+#### Configuring the connector in the Airbyte UI
 1. Toggle whether your Salesforce account is a Sandbox account or a live account.  
 2. Click `Authenticate your Salesforce account` to sign in with Salesforce and authorize your account.
 3. Fill in the rest of the details.
 4. You should be ready to sync data. 
 
 ## Getting started (Airbyte OSS) 
 ### Requirements
-* Salesforce Account
-* Salesforce OAuth credentials
-* Dedicated Salesforce user (optional)
-
-**Note**: We recommend creating a new Salesforce user, restricted, read-only OAuth credentials specifically for Airbyte access. In addition, you can restrict access to only the data and streams you need by creating a profile in Salesforce and assigning it to the user.
-
-### Setup guide
+* Salesforce Account with Enterprise access or API quota purchased
+* Dedicated read only Salesforce user (optional)
+* Salesforce OAuth credentials 
 
 #### Sandbox accounts
 If you log in using at [https://login.salesforce.com](https://login.salesforce.com), then your account is not a sandbox. If you log in at [https://test.salesforce.com](https://test.salesforce.com) then it's a sandbox. 
 
 If this is Greek to you, then you are likely not using a sandbox account.
+
+### Setup guide
+
+#### Read only user
+See the [section above](#creating-a-dedicated-read-only-salesforce-user) for creating a read only user. This step is optional.  
+
+#### Salesforce Oauth Credentials
 We recommend the following [walkthrough](https://medium.com/@bpmmendis94/obtain-access-refresh-tokens-from-salesforce-rest-api-a324fe4ccd9b) **while keeping in mind the edits we suggest below** for setting up a Salesforce app that can pull data from Salesforce and locating the credentials you need to provide to Airbyte.
 
 Suggested edits:
 
 1. If your salesforce URL does not take the form `X.salesforce.com`, use your actual Salesforce domain name. For example, if your Salesforce URL is `awesomecompany.force.com` then use that instead of `awesomecompany.salesforce.com`. 
 2. When running a `curl` command, always run it with the `-L` option to follow any redirects.
+3. If you created a read only user, use those credentials when logging in to generate oauth tokens
 
 ## Streams
 
-**Note**: The connector supports reading both standard streams and Custom Objects from Salesforce.
+**Note**: The connector supports reading both Standard Objects and Custom Objects from Salesforce. Each object is read as a separate streams. 
```

Comment:
"read as a separate streams." shouldn't have an 's' for stream.

Reply:
i am edumucated very good